### ISSUE TITLE
Game Physics: Awkward Phrasings

### ISSUE THREAD

**Issue by Mornix** [smiths/caseStudies#48]
(Issue #29)

The following is a list of awkward phrasings within the [Game Physics SRS](https://github.com/smiths/caseStudies/blob/master/CaseStudies/gamephys/documentation/SRS/GamePhysicsSRS.pdf):
- [ ] **Section 2.3**'s first sentence is incomplete and reads awkwardly. "Reviewers of this documentation should have a strong knowledge, which is at the level covered in a second year engineering mechanics course." 
- [ ] **Section 4.1** has a sentence (3rd sentence) which reads "Developing a physics library from scratch takes a long period of time and is very costly, presenting barriers of entry which make it difficult for game developers to include physics in their products." Games which don't have any form of "physics" seems slim. Maybe a more accurate description would be: "...which make it difficult for game developers to include **realistic** physics in their products."
- [ ] **Section 4.1.1 (Elasticity)** has "... after and before the collision." which sounds awkward.
- [ ] **Section 4.2.5 (IM3's Description)** has the sentence (when describing _t_ and _t0_) "...denotes the time at collision (s)," which sounds awkward.

**Comment by smiths** [comment]
@Mornix, I agree with your criticism.  Can you please replace the awkward phrasings with improved phrasing.  You can then create an issue for me to review your revised wording.  Once we settle on the changes, we will have to propagate this information to the Drasil repo.
